Gibe.Umbraco.Blog/BlogService.cs

```diff
@@ -86,7 +86,9 @@ public T GetPreviousPost(T current, IEnumerable<IBlogPostFilter> filters, ISort
 
 		public IEnumerable<T> GetRelatedPosts(T post, int count)
 		{
-			throw new NotImplementedException();
+			var filter = new TagsBlogPostFilter(post.Tags);
```

Comment:
Is `Filter` behaviour what we want here? If I give you a post on `#shopping #kittens #pink`, do we not want to return posts about `#shopping #kittens #blue`?

Reply:
This is the solution that @stevetemple and I decided to use, this should use a groupedOr query and therefore hopefully would return everything with any of the requested tags rather than actually filtering out anything that is missing one of said tags.

Reply:
Ah! Cool.

Reply:
Although it's a breaking change it might be worth considering in future renaming the filter to something that doesn't imply it's all these tags

SimilarTagsFilter
AtLeastOneMatchingTagFilter